controller_manager/controller_manager/controller_manager_services.py

```diff
@@ -32,7 +32,39 @@ class ServiceNotFoundError(Exception):
     pass
 
 
-def service_caller(node, service_name, service_type, request, service_timeout=0.0):
+def service_caller(
+    node,
+    service_name,
+    service_type,
+    request,
+    service_timeout=0.0,
+    call_timeout=10.0,
```

Comment:
@fmauch one more question, do you see an easy way to make these configurable by the user? My bringup is quite CPU intensive causing the call_timeout to be reached, however it works if I increase it a bit

Reply:
If not, any harm in arbitrarily increasing it to e.g. 30?

Reply:
https://github.com/ros-controls/ros2_control/pull/1808